apps/client/src/routes/dashboards/create/create-dashboard-page.tsx

```diff
@@ -26,39 +27,43 @@ export function CreateDashboardPage() {
   }
 
   return (
-    <form
-      onSubmit={(event) => {
-        event.preventDefault();
+    <>
+      <form
+        onSubmit={(event) => {
+          event.preventDefault();
 
-        void handleSubmit((formData) => {
-          createDashboardMutation.mutate(formData);
-        })();
-      }}
-    >
-      <Form
-        variant="full-page"
-        header={
-          <Header variant="h1">
-            <FormattedMessage
-              defaultMessage="Create dashboard"
-              description="create dashboard heading"
-            />
-          </Header>
-        }
-        actions={
-          <CreateDashboardFormActions
-            isLoading={createDashboardMutation.isLoading}
-          />
-        }
-        errorText={getFormErrorText()}
+          void handleSubmit((formData) => {
+            createDashboardMutation.mutate(formData);
+          })();
+        }}
       >
-        <Container>
-          <SpaceBetween size="l">
-            <DashboardNameField control={control} />
-            <DashboardDescriptionField control={control} />
-          </SpaceBetween>
-        </Container>
-      </Form>
-    </form>
+        <Form
+          variant="full-page"
+          header={
+            <Header variant="h1">
+              <FormattedMessage
+                defaultMessage="Create dashboard"
+                description="create dashboard heading"
```

Comment:
Is this meant to be all lowercase? Would've expected "Create dashboard heading" for the casing.

Reply:
All of our descriptions have been created as all lowercase. What is the benefit to using sentence case for the message descriptors?